src/v2/models/AppState.js

```diff
@@ -153,6 +153,12 @@ export default Model.extend({
       && !FORMS_WITHOUT_SIGNOUT.includes(currentFormName);
   },
 
+  isExpiredEmailLinkTerminalState () {
```

Comment:
if we can pass a `key` as argument then we can make this function generalized. I won't prefer a function that does only one thing. Please use an argument here.